interface/patient_file/report/custom_report.php

```diff
@@ -823,7 +822,9 @@ function next_prev(action){
                     '/documents/' . $from_pathname . '/' . $from_filename;
                   $to_file = substr($from_file, 0, strrpos($from_file, '.')) . '_converted.jpg';
                 }
-
+                //Extract the extension by the mime/type and not the file name extension
+                $image_data = getimagesize($from_file);
+                $extension = image_type_to_extension($image_data[2]);
```

Comment:
Just confirm for me that this fix is testing well for you with and without couchdb; then I'll bring it into the main codebase.(also, in future, ensure use the pull request on the openemr/openemr repo rather than my personal repo)